### DISCUSSION TITLE
Enable Image Insertion in base 64 in CKEDITOR

### DISCUSSION THREAD

**Discussion by emmanuelpineda** [Lumieducation/H5P-Nodejs-library#2141] [Q&A]
First of all, I would like to thank you for the great job you have done!

I am using this component "H5PEditorUI" from this library "@lumieducation/h5p-react".

![image](https://user-images.githubusercontent.com/45979790/157933137-f14ed28a-31bc-46d3-831f-0cf8aa23b320.png)

Is it possible to enable image insertion in base 64?

I have done some research, it seems that it is necessary to add a plugin to ckeditor, how can I do it using the "@lumieducation" libraries.

Thank you again!

**Comment by sr258** [comment]
@emmanuelpineda This cannot be done in our libraries. It would require a modified H5P editor core that changes what CKEditor display. I think the place to do it is here: https://github.com/h5p/h5p-editor-php-library/blob/master/scripts/h5peditor-html.js

As an application using our libraries has to include this file anyway, you can simply patch it for your application.
